sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Tests.csproj

```diff
@@ -22,7 +22,6 @@
     <ProjectReference Include="$(AzureCoreTestFramework)" />
     <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
     <ProjectReference Include="..\..\..\eventhub\Azure.Messaging.EventHubs\src\Azure.Messaging.EventHubs.csproj" />
```

Comment:
This is temporary due to the current set of changes?   Is it safe to assume that we'll shift to a package reference once things are released?

Reply:
Yes we can do that.

Reply:
Ah, I misread and didn't realize this was the test library.   No objection to leaving as-is.